fabtests/scripts/runmultinode.py

```diff
@@ -24,7 +24,7 @@ def parse_args():
 
 def parse_config(config):
 	with open(config, "r") as f:
-		yconf = yaml.load(f, Loader=yaml.FullLoader)
+		yconf = yaml.safe_load(f)
```

Comment:
Have you verified that the default loader for the `safe_load` method (`SafeLoader`) work fine here?  There might be some reason behind the use of `FullLoader` in the original code.

Reply:
No. I just want to make sure that the capability SafeLloader is sufficient for the configuration files used for fabtests.

I think we can go with the default safe loader for now and change to FullLoader if that is discovered to be necessary.